e2e/config-test.toml

```diff
@@ -0,0 +1,120 @@
+[general]
+# apiEndpoint = "http://localhost:8090"
+# apiEndpoint = "http://127.0.0.1:8090"
+# apiEndpoint = "http://10.100.11.1:8090"
+# apiEndpoint = "https://dogbowl.cloud.backend.ai"
+# apiEndpoint = "http://10.82.230.101:8090"
+# apiEndpoint = "http://10.100.64.15:8090"
+# apiEndpoint ="https://aitrain.ktcloud.com"
+
+
+# apiEndpoint = "http://10.82.230.94:8090"
+# apiEndpoint = "http://10.82.230.101:8090"
+# apiEndpoint = "https://cloud.backend.ai"
+#apiEndpoint = "http://10.82.230.115:8090"
+# apiEndpoint = "http://10.172.160.61:8080"
+# apiEndpoint = "http://10.82.230.230:8090"
+# apiEndpoint = "https://www.nipa2023.ktcloud.com:8000"
+# apiEndpoint = "https://cloud.backend.ai"
+# apiEndpoint = "http://0.0.0.0:8090"
+# apiEndpoint = "http://10.100.64.13:8090"
+# apiEndpoint = "http://10.20.30.10:9004"
+apiEndpointText = "Backend.AI Admin Training"
+# defaultSessionEnvironment = "cr.backend.ai/testing/mojo:2309"
+# defaultSessionEnvironment = "cr.backend.ai/stable/python"
+defaultSessionEnvironment = "cr.backend.ai/stable/python-tensorflow"
+
+# defaultSessionEnvironment = "cr.backend.ai/community/spectravis:19.10"
+maxCountForPreOpenedPort = 1
+# defaultSessionEnvironment = "cr.backend.ai/stable/python:3.8-ubuntu20.04-arm64"
+
+#defaultSessionEnvironment = "[Default session kernel. If blank, alphabetically first kernel will be default.]"
+# defaultImportEnvironment = "cr.backend.ai/stable/python:3.8-ubuntu20.04-arm64"
+# defaultImportEnvironment = "cr.backend.ai/multiarch/python:3.9-ubuntu20.04-aarch64"
+# defaultImportEnvironment = "cr.backend.ai/multiarch/python:3.9-ubuntu20.04"
+# defaultImportEnvironment = "cr.backend.ai/stable/python"
+siteDescription = "NIPA NPU Farm"
+connectionMode = "SESSION" #connectionMode = API
+allowChangeSigninMode = true
+signupSupport = true
+allowSignout = true
+allowAnonymousChangePassword = true
+allowProjectResourceMonitor = true
+allowManualImageNameForSession = true
+allowSignupWithoutConfirmation = true
+allowCustomResourceAllocation = false
+autoLogout = false
+# loginAttemptLimit = 1
+# loginBlockTime = 30
+debug = false
+maskUserInfo = true
+enableContainerCommit = true
+enablePipeline = true
+hideAgents = true
+alwaysEnqueueComputeSession = true
+# If false, show the `Agent Summary` menu in the sidebar.
+showCustomResourceAllocation = true
+enable2FA = true
+force2FA = false
+# systemSSHImage = "cr.backend.ai/testing/filebrowser:23.04-ubuntu20.04-arm64"
+# systemSSHImage = "cr.backend.ai/testing/filebrowser:23.10-amd64"
+# cr.backend.ai/testing/filebrowser:23.04-ubuntu20.04
+isDirectorySizeVisible = false
+allowCustomResourceAllocation = true
+forceTerminateTimeThreshold = 259200
+enableModelStore = true
+enableExtendLoginSession = false
+showNonInstalledImages = false
```

Comment:
`showNonInstalledImages` is a configuration in `environments` section, right? Please delete this line.